torch/csrc/jit/export.cpp

```diff
@@ -536,6 +538,9 @@ class ScriptModuleSerializer final {
 
   std::vector<IValue> attribute_table_;
 
+  // result of module __getstate__
```

Comment:
I think there should only be one table and one pkl file. Otherwise cross references between objects will get broken. Can you add a test for this as well?